src/store/enums.ts

```diff
@@ -0,0 +1 @@
+export * from './branches/auth/enums';
```

Comment:
Oh, so you've taken a different approach here? Why don't we apply it to the rest of the files - `interfaces.ts` too for example?

Reply:
It's applied, it's just we don't import the interfaces outside the store on this phase